flink-runtime/src/test/java/org/apache/flink/runtime/io/network/partition/hybrid/tiered/netty/TieredStorageResultSubpartitionViewTest.java

```diff
@@ -115,18 +113,18 @@ void testNotifyRequiredSegmentId() {
     @Test
     void testReleaseAllResources() throws IOException {
         tieredStorageResultSubpartitionView.releaseAllResources();
-        assertThat(nettyPayloadQueues.get(0)).hasSize(0);
-        assertThat(nettyPayloadQueues.get(1)).hasSize(0);
+        assertThat(nettyPayloadQueues.get(0).getBacklog()).isEqualTo(0);
```

Comment:
```suggestion
        assertThat(nettyPayloadQueues.get(0).getBacklog()).isZero();
```

Reply:
Done, I have modified all occurrence of `isEqualTo(0)` to `isZero()`.